test/1_test_fullpath.js

```diff
@@ -940,13 +938,13 @@ contract('Voucher tests', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey2
       );
 
       //[1001.0000] = hex"90" = 144 = EXPIRED
       assert.equal(
-        web3.utils.toHex(voucherStatus.status),
+        web3.utils.toHex(voucherStatus[0]),
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)